python/vyos/util.py

```diff
@@ -191,7 +191,6 @@ def read_file(fname, defaultonfailure=None):
         """ Read a file to string """
         with open(fname, 'r') as f:
             data = f.read().strip()
-        return data
```

Comment:
Why no longer returning the data we have read? I guess this will break all kind of stuff.

Reply:
Wow... What a stupid mistake. Thanks, fixed.